sos/report/plugins/openstack_aodh.py

```diff
@@ -88,4 +87,19 @@ def postproc(self):
         )
 
 
+class DebianAodh(OpenStackAodh, DebianPlugin, UbuntuPlugin):
```

Comment:
The preferred style is to use the full previous class name, e.g. this would be `DebianOpenStackAodh`

Reply:
Done.